src/page.ts

```diff
@@ -373,16 +373,30 @@ export class Page extends EventTarget {
    */
   async cookies(...urls: string[]): Promise<Cookie[]> {
     const result = await retryDeadline(
-      this.#celestial.Network.getCookies({ urls }),
+      this.#celestial.Network.getCookies({
+        urls: urls.length ? urls : undefined,
+      }),
```

Comment:
What does this fix?

Reply:
Before this fix, if you try the method without specifying arguments `page.cookies()`, it will return an empty array

```typescript
import { launch } from "./mod.ts";
const browser = await launch();
const page = await browser.newPage("https://deno.land");
console.log(await page.cookies());  // []
await browser.close();
```

With this fix, if you apply the same example, 2 cookies will appear.